qnm_filter/utility.py

```diff
@@ -42,7 +43,45 @@ def parallel_compute(self, M_arr, chi_arr, num_cpu=-1, **kwargs):
         delayed(self.likelihood_vs_mass_spin)(i, j, **kwargs) for i, j in flatten_array
     )
     reshaped_results = np.reshape(results, (len(M_arr), len(chi_arr))).T
-    return reshaped_results
+    return reshaped_results, logsumexp(reshaped_results)
+
+
+def evidence_parallel(
+    self, index_spacing, num_iteration, M_arr, chi_arr, num_cpu=-1, **kwargs
```

Comment:
This always starts the evidence_parallel from the initial value of i0, would it be better to specify three arguments: index_spacing, num_iteration, initial_offset? When we chunk up the t_init space we want to calculate the evidence curve for, this would allow for more readable code I think

Reply:
Done